models/db.js

```diff
@@ -6,7 +6,7 @@
 var mongoose = require('mongoose');
 var Schema = mongoose.Schema;
 
-mongoose.connect(process.env.MONGOLAB_URI);
+mongoose.connect(process.env.MONGOLAB_URI || "mongodb://localhost:27017/mongodb");
```

Comment:
I wrote about this before.  Change back to `mongoose.connect(process.env.MONGOLAB_URI);`